### PR TITLE
Feature/helper fns

### PR DESCRIPTION
**Pull request by dpetrovics**
Two cljs time helper functions, and replace-index, with tests

### CODE REVIEW

File: src/cljx/racehub/util/time.cljx [hunk 1]
```diff
@@ -345,16 +345,33 @@
...
          (format/parse (format/formatter "MM/dd/yyyy"))
          (coerce/to-long)))
 
+  (s/defn unix->calendar-str :- s/Str
+    "Converts the given UnixTime to a string of format mm/dd/yyyy."
+    [unix-time :- UnixTime]
+    (format/unparse (format/formatter "MM/dd/yyyy")
+                    (local-js-date-from-unix unix-time)))
+
+  (s/defn to-display-str :- s/Str
```
Comment (dpetrovics) [inline src/cljx/racehub/util/time.cljx]: this guy i just moved up from below

File: src/cljx/racehub/util.cljx [hunk 2]
```diff
@@ -155,6 +155,20 @@
                [element]
                (subvec v index (count v)))))
 
+(s/defn replace-index :- [s/Any]
+  "Replaces the item at the given index of the collection with the new
+  item. For lists only, if an index is smaller then 0, appends to the
+  beginning - if the index is greater than the (dec count), it'll
```
Comment (sritchie) [inline src/cljx/racehub/util.cljx]: is this still true if you pass in a vector?

File: test/clj/racehub/util_test.clj [hunk 3]
```diff
@@ -223,3 +223,17 @@
     (is (= (build-kw-map d) {:d {:foo "bar" :big {:dawg "33"}}}))
     (is (= (build-kw-map b f) {:b 23 :f :yomama}))
     (is (= (build-kw-map a b c e) {:a "Hi" :b 23 :c ["A" 43] :e #{:hi :there}}))))
+
+(deftest replace-index-test
```
Comment (sritchie) [inline test/clj/racehub/util_test.clj]: This is a perfect place for some generative testing :)

**Comment by sritchie** [comment]
Can you squash this into one commit?

**Comment by sritchie** [comment]
Otherwise, looks fine, go for it.

**Comment by dpetrovics** [comment]
closed in favor of #31

**Pull request closed without merging**